app/server.go

```diff
@@ -103,6 +103,10 @@ func RunServer(c *Config) error {
 			CronTab: "0/20 * * * *",
 			Func:    appealJobHandler.RevokeExpiredAccess,
 		},
+		{
+			CronTab: "0 9 * * *", // at 09.00
```

Comment:
will this configurable as well in the next update?

Reply:
yes